### PR TITLE
adding validation to pulled json

### PR DESCRIPTION
**Pull request by gvn**
**DON'T MERGE YET** – Django "People" data is currently malformed.

### CODE REVIEW

**Comment by gvn** [comment]
This will validate the JSON being pulled from Django as a test as well as validating that `npm build` succeeds as a test.

File: source/json/people.schema.json [hunk 1]
```diff
@@ -9,8 +9,7 @@
         "chance": "name"
       },
       "role": {
-        "type": "string",
-        "pattern": "Staff|OWF Host|Partner|Hive NYC Partner|Advisor|Member, MLN Advisory Group"
```
Comment (gvn) [inline source/json/people.schema.json]: @gideonthomas @alanmoo – I had to remove this pattern to get the Django-returned JSON to validate. Is `role` being treated as a freeform string in Django? If so, is that going to be problematic for any future front-end functionality you're aware of, @alanmoo ?

Comment (gideonthomas) [inline source/json/people.schema.json]: I can fix that if we have a specified list of acceptable roles and we want to confine it to those
cc @hannahkane

Comment (gvn) [inline source/json/people.schema.json]: Based on stand conversation: Roles will be freeform.

**Pull request merged**